### PR TITLE
Add centos 8

### PR DESCRIPTION
**Pull request by SMillerDev**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,6 +24,11 @@ docker build -t linuxbrew-centos6 https://raw.githubusercontent.com/Linuxbrew/do
 docker build -t linuxbrew-centos7 https://raw.githubusercontent.com/Linuxbrew/docker/master/centos7/Dockerfile
 ```
 
+## CentOS 8
+```sh
+docker build -t linuxbrew-centos7 https://raw.githubusercontent.com/Linuxbrew/docker/master/centos8/Dockerfile
```
Comment (issyl0) [inline README.md]:
```suggestion
docker build -t linuxbrew-centos8 https://raw.githubusercontent.com/Linuxbrew/docker/master/centos8/Dockerfile
```

File: README.md [hunk 2]
```diff
@@ -56,10 +61,10 @@ docker build -t linuxbrew-trusty https://raw.githubusercontent.com/Linuxbrew/doc
...
 ## Ubuntu 16.04 LTS (Xenial Xerus)
 ```sh
-docker run -it linuxbrew/linuxbrew
+docker build -t linuxbrew-xenial https://raw.githubusercontent.com/Linuxbrew/docker/master/xenial/Dockerfile
 ```
 
 ## Ubuntu 18.04 LTS (Bionic Beaver)
 ```sh
-docker run -it linuxbrew/linuxbrew
+docker run -it linuxbrew/brew
```
Comment (issyl0) [inline README.md:69]: _Technically_ this is now Homebrew/brew, but I'm not sure how maintained this repo is in general so maybe this particular nitpick is out of scope.

**Review by issyl0 — APPROVED** [review]
Travis CI looks broken. All it does it run `dockerfilelint`. I'll look into porting it to GitHub Actions tomorrow.

Later on in my review there's an existential question about the updatedness of this repo in general, but this addition looks good.

Thanks, Sean!

**Comment by issyl0** [comment]
That linting error is pre-existing and unrelated to these changes. I don't know what - if anything - uses that Dockerfile. Going to ignore it for the purposes of this. Thanks again, @SMillerDev!

File: README.md (hunk 2 above)
Comment (sjackman) [reply README.md:69]: `homebrew/brew` is preferred over `linuxbrew/brew`.

Comment (sjackman) [reply README.md:69]: This line should be
```dockerfile
docker build -t linuxbrew-bionic https://raw.githubusercontent.com/Linuxbrew/docker/master/bionic/Dockerfile
```

Comment (sjackman) [reply README.md:69]: `homebrew/brew` is based on Ubuntu 16.04 LTS (Xenial Xerus).
